document-store/src/main/java/org/hypertrace/core/documentstore/expression/Aggregator.java

```diff
@@ -0,0 +1,7 @@
+package org.hypertrace.core.documentstore.expression;
+
+public enum Aggregator {
+  COUNT,
```

Comment:
distinct count is often more interesting than count and many of our use cases are built on it - probably want that too.

Reply:
+1